src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1497,6 +1506,21 @@ namespace UnitarySystems {
             select_EqSizing = &OASysEqSizing(state.dataSize->CurOASysNum);
         } else if (state.dataSize->CurSysNum > 0) {
             select_EqSizing = &state.dataSize->UnitarySysEqSizing(state.dataSize->CurSysNum);
+            // this was reseting data set by OutdoorAirUnit when UnitarySystem is child
+            // question is then who resets these (#8751 temporary fix)?
+            // move here for now and only reset UnitarySystem flags, then find better way to do this
+            select_EqSizing->AirFlow = false;
+            select_EqSizing->CoolingAirFlow = false;
+            select_EqSizing->HeatingAirFlow = false;
+            select_EqSizing->AirVolFlow = 0.0;
+            select_EqSizing->CoolingAirVolFlow = 0.0;
+            select_EqSizing->HeatingAirVolFlow = 0.0;
+            select_EqSizing->Capacity = false;
+            select_EqSizing->CoolingCapacity = false;
+            select_EqSizing->HeatingCapacity = false;
+            select_EqSizing->DesCoolingLoad = 0.0;
+            select_EqSizing->DesHeatingLoad = 0.0;
+            select_EqSizing->OAVolFlow = 0.0; // UnitarySys doesn't have OA
```

Comment:
This general reset was zeroing globals set up by OutdoorAirUnit. I think the parent should do the reset, set these up, then size all children. The next parent will reset these.